documentation/docs/getting_started.md

```diff
@@ -1,14 +1,34 @@
 ---
-description:   Choose your binding and get started with the IOTA Client Library. 
+description:   Choose your binding and get started with the IOTA Client Library.
 image: /img/logo/iota_mark_light.png
 keywords:
 - rust
 - python
 - node.js
+- java
+- javascript
 - IDE
+- reference
 ---
 # Getting Started
 
-To check out our examples and write and test your own code, you would need an IDE or a code editor of your choice and a stable internet connection. You also need to set up your environment by following instructions for one of the languages: [Rust](./libraries/rust/getting_started.md), [Node.js](./libraries/nodejs/getting_started.md), [Python](./libraries/python/getting_started.md), or [Java](./libraries/java/getting_started.md).
+## Required Prior Knowledge
 
-We assume that you already know the basics of the programming language that you chose. Companies and communities behind each language provide their own documentation for beginners: [Rust](https://www.rust-lang.org/learn/get-started), [JavaScript](https://www.w3schools.com/js/) and [Node.js](https://nodejs.org/en/docs/guides/), [Python](https://www.python.org/about/gettingstarted/), [Java](https://www.java.com/). You could start with that first, or refer to these guides as you read through our iota.rs documentation. If you have never programmed in your life, MIT has published an open [introductory course to programming](https://ocw.mit.edu/courses/intro-programming/#general). Check it out!
\ No newline at end of file
+To check out our examples and write and test your own code, you will need an IDE or a code editor of your choice and a
+stable internet connection. You also need to set up your environment by following instructions for one of the
+languages: [Rust](./libraries/rust/getting_started.md), [Node.js](./libraries/nodejs/getting_started.md)
+, [Python](./libraries/python/getting_started.md), or [Java](./libraries/java/getting_started.md).
+
+This wiki assumes that you already know the basics of your programming language of choice. Companies and communities
+behind each language provide their documentation for beginners you can use as reference:
+
+* [Java](https://www.java.com/)
+* [JavaScript](https://www.w3schools.com/js/)
+* [Node.js](https://nodejs.org/en/docs/guides/)
+* [Python](https://www.python.org/about/gettingstarted/)
+* [Rust](https://www.rust-lang.org/learn/get-started)
+
+If you have little experience coding, you could start with any of those first or refer to these guides as you read
```

Comment:
Oh, I remember this, I have written the previous version. I will exclude myself from discussing this part.